addons/web/static/src/views/fields/properties/properties_field.js

```diff
@@ -323,13 +344,14 @@ export class PropertiesField extends Component {
      * if we don't have access for parent
      */
     async checkDefinitionWriteAccess() {
-        const definitionRecordId = this.props.record.data[this.definitionRecordField][0];
-        const definitionRecordModel = this.props.record.fields[this.definitionRecordField].relation;
+        if (!this.definitionRecordId || !this.definitionRecordModel) {
```

Comment:
In which use case do we not have these?

Reply:
Would update docstring then (if we don't have access to parent or don't find any parent, for example)